dmf/scripts/mods/dmf/modules/ui/options/mod_options.lua

```diff
@@ -350,10 +350,58 @@ local function create_option_template(self, widget_data, category_name, index_of
   end
 end
 
+-- Insert a new item into a table before any items that pass the item_tester function
+local function insert_before(tbl, item_tester, new_item)
+	local copy = {}
+	for _, item in ipairs(tbl) do
+		if item_tester(item) then
+			table.insert(copy, new_item)
+		end
+		table.insert(copy, item)
+	end
+	return copy
+end
+
+
 -- ####################################################################################################################
 -- ##### Hooks ########################################################################################################
 -- ####################################################################################################################
 
+-- Add Mods Options title to global localization table
+-- so that the SystemView options menu can localize it
+local localization_table = dmf:io_dofile("dmf/localization/dmf")
```

Comment:
With bundling on the horizon, this will become a problem. I'd probably just copy the strings here until we have a better method.

Reply:
Gotcha, done!